pybloom/pybloom.py

```diff
@@ -95,7 +97,7 @@ def _make_hashfuncs(key):
                 if i >= num_slices:
                     return
 
-    return _make_hashfuncs
+    return _make_hashfuncs, hashfn_name
```

Comment:
Do the variable names ( `_make_hashfuncs` and `hashfn_name`) seem a bit confusing here - the singular and plural nouns seem a bit contradict to each other? 

`make_hashfuncs` essentially returns a hash maker.  I think we could do a bit name refactoring here to reflect the fact that one is hash maker and the other is the hash function used by the hash maker.